assembly/pom.xml

```diff
@@ -94,7 +94,8 @@
                                     <exclude>com.splicemachine:db-tools-ij</exclude>
                                     <exclude>com.splicemachine:db-shared</exclude>
                                     <exclude>com.splicemachine:db-drda</exclude>
-                                    <exclude>${extraExclude}</exclude>
+                                    <exclude>${excludeSpark}</exclude>
+                                    <exclude>${excludeKafka}</exclude>
```

Comment:
Shouldn't Kafka be excluded for all platforms, not just CDH?

Reply:
Yes, will make changes